backend/testfiles/execution/language.tests

```diff
@@ -32,27 +32,49 @@ module Records =
   { col1 = 2 ; col2 = Test.typeError_v0 "test" } = Test.typeError_v0 "test"
   { col1 = 2 ; ___ = 3 } = Test.typeError "Record key is empty"
 
+
 module UserEnums =
```

Comment:
am I missing classes of basic obvious tests here?

Reply:
Pattern matching for variants of different sizes I think